meinberlin/apps/projects/tasks.py

```diff
@@ -190,6 +219,12 @@ def reset_cache_for_projects(starts: bool, ends: bool) -> str:
         ):
             msg += "failed for active projects becoming past"
         else:
-            msg += "succeeded for active projects becoming past"
-    logger.info(msg)
-    return msg
+            msg += "succeeded for future projects becoming active"
+
+    # set the redis key: value for project's queryset
+    try:
+        cache.set("project_queryset", get_project_queryset())
```

Comment:
I think this might not work because the queryset at this point is not evaluated yet, I remember having that issue when I looked into it, that's why I had to cache it after serializing it.